src/coord/rmr/rq.c

```diff
@@ -124,7 +125,7 @@ static void verify_uv_thread() {
     uv_async_init(uv_default_loop(), &topologyAsync, topologyAsyncCB);
     // Verify that we are running on the event loop thread
     int uv_thread_create_status = uv_thread_create(&loop_th, sideThread, NULL);
-    RedisModule_Assert(uv_thread_create_status == 0);
+    RS_ASSERT(uv_thread_create_status == 0);
```

Comment:
I'm not sure if this is aligned in all versions, just so you are aware. In previous versions, the call to `uv_thread_create` may still be in the assertion expression, so we must move it out like here